eng/common/scripts/Verify-Links.ps1

```diff
@@ -134,6 +136,10 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 
 function CheckLink ([System.Uri]$linkUri)
 {
+  if(!$linkUri.ToString().Trim()) {
```

Comment:
What happens if we don't check this here and let it fall through to our other checks? Will it correctly fail as a relative path error?

Reply:
We have to check it in relative link section with more if blocks to avoid null pointer exception.

I intended to put it here to avoid all checking below as it doesn't make too much sense to evaluate empty link.

Reply:
If it is more trouble then it is worth then I'm fine with special casing it here. However in general if we can avoid the special cases it make things less complex and more maintainable.